lib/codecept.js

```diff
@@ -73,6 +73,7 @@ class Codecept {
       global.within = require('./within');
       global.session = require('./session');
       global.DataTable = require('./data/table');
+      global.DataTableArgument = require('./data/dataTableArgument');
```

Comment:
do we really need it here? 

If you want to use DataTableArgument as a wrapper to table from Cucumber for the way you listed in docs, it is better to add one into `lib/index.js` to export so you could use it like that:

```
comst { DataTableArgument } = require('codeceptjs');
  const dataTableArgument = new DataTableArgument(table);
```

so it won't pollute global namespace and core file `codecept.js`

Reply:
I agree. Removed from the global namespace.
Already exists in the `lib/index.js`